config/application.rb

```diff
@@ -42,6 +43,18 @@ class Application < Rails::Application
       ENV[
         "DATABASE_URL"
       ] = "postgres://#{username}:#{password}@#{host}:#{port}/#{dbname}"
+      # for environment which uses RDS aurora managed credentials only the the username
+      # and password is automaticallyset. The environment variable is then DB_CREDENTIALS
```

Comment:
Would it make sense to have these comments after the `elsif`? I can see they are related to that block because it's obvious in this diff, but if I were looking at this outside the diff I'm not so sure it would be as clear.